src/lib/responder.ts

```diff
@@ -4,11 +4,16 @@ import responseFactory from './response-triggers/index';
 import __logger from './logger';
 
 async function respond(messageEvent : {data : {id : string}}) {
-	try {
+	try {		
 		let message : MessageObject = await spark.messages.get(messageEvent.data.id);
 		__logger.debug('processing message: ' + JSON.stringify(message));
 		let room = message.roomId;
 
+		if (message.personEmail.includes('@sparkbot.io') || message.personEmail.includes('@webex.bot')) {
```

Comment:
Should be using `person.type === `type.bot` or however the enum comparison works.